doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -47,7 +47,7 @@ In addition, note that a list of currently open discussions is available at
 
 Copyright (c) 2011-2016 Telecom Italia, Telefónica I+D and NEC. 
 
-## Legal Notice
+## License
 
 This specification is licensed under the
 [FIWARE Open Specification License (implicit patent license)]
```

Comment:
There is a render problem related to:

```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```
FABRE cannot detect links if you insert a line break between `]` and `(`. So, the previous text should be something like (please note that there is no like break between `]` and `(` ):
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```

If you need to break the line, please place it in another position. You can even use multiplies break lines: 
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit 
patent license)](https://forge.fiware.org/plugins/
mediawiki/wiki/fiware/index.php/Implicit_Patents_License).

```
You can use http://dillinger.io/ to test different places where a line break can be inserted inside a link.